core/src/trezor/ui/layouts/delizia/__init__.py

```diff
@@ -39,7 +39,7 @@ def confirm_action(
     br_code: ButtonRequestType = BR_CODE_OTHER,
     prompt_screen: bool = False,
     prompt_title: str | None = None,
-) -> Awaitable[None]:
+) -> Awaitable[ui.UiResult | None]:
```

Comment:
IIUC, the `| None` above is due to the use of `raise_if_cancelled()` here:
https://github.com/trezor/trezor-firmware/blob/9e23d54f47253c66fcc0994c3911cfb54f7721ae/core/src/trezor/ui/layouts/delizia/__init__.py#L66

Maybe we can use `interact()` instead - to simplify the return type of this function?
(also, we don't really return `None` in this case...)

Reply:
See recent discussion here: https://github.com/trezor/trezor-firmware/pull/5349 :)